### PR TITLE
Added Domain in Excluded Global variable

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
Added below domains in Excluded List Global Variable

old domain list - "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com"

Updated Domain List: 'google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com'

Changes Made:

> Updated -> 11 Demo >> Create Default Global Variables playbook
> Updated -> Global Variables > 'Excludelist_Domains'

### CODE REVIEW

**Comment by cs-suyog-jinturkar** [comment]
What all unit testing was carried out to test this change?

**Comment by cs-rohit-khune** [comment]
Refer to attached Unit Test
[Mantis ID - 0770430 Unit Test Case Document.docx](https://github.com/fortinet-fortisoar/solution-pack-incident-response/files/7912210/Mantis.ID.-.0770430.Unit.Test.Case.Document.docx)

File: playbooks/globalVariables.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     {
         "id": 11,
         "name": "Excludelist_Domains",
-        "value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com",
-        "default_value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com"
+        "value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com",
+        "default_value": "google.com,yahoo.com,fortinet.net,cybersponse.com,gmail.com,outlook.com,microsoft.com,fortinet.com,twitter.com,facebook.com,linkedin.com,instagram.com"
```
Comment (cs-tushar-kanade) [inline playbooks/globalVariables.json]: add fortiguard.com , forticloud.com  as well

**Review by cs-tushar-kanade — CHANGES_REQUESTED** [review]
Please do add requested items

File: playbooks/globalVariables.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/globalVariables.json]: @cs-pooja-singh @cs-tushar-kanade Added fortiguard.com , forticloud.com in globalVariable.json and Create Default Global Variable.json

**Review by cs-tushar-kanade — APPROVED** [review]

**Comment by cs-tushar-kanade** [comment]
@cs-suyog-jinturkar please check and merge

**Pull request merged**